### DISCUSSION TITLE
COPOD - standardize variables?

### DISCUSSION THREAD

**Discussion by akarpf** [yzhao062/pyod#486] [General]
Hi! Do I need to standardize the variables before using copod? In the COPOD example copod_interpretability.py it's done, but in my opinion it shouldn't make a difference, no? The ECDF should give the same results, no matter if the variable is standardized or not. 

I would be very happy to get some input from you on this question. Thanks!

**Comment by yzhao062** [comment]
that is correct. no diff

**Reply by akarpf** [reply]
Thank you!
